### ISSUE TITLE
Management endpoints for blockchain wallets

### ISSUE THREAD

**Issue by T-Dnzt** [omisego/ewallet#691]
Depends on #690 

In order to manage the blockchain wallets, the following endpoints need to be added to the Admin API.

- `/blockchain_wallet.get`
- `/blockchain_wallet.all`

**Comment by unnawut** [comment]
What about reusing `/wallet.create` etc. with `blockchain: true`? Otherwise from user point of view everything will need to be handled twice with local wallets and blockchain wallets?

**Comment by T-Dnzt** [comment]
Yes, they should be handled separately. You may have the following structure:

- wallet 1 (100) -----\
- wallet 2 (234) -----| Blockchain wallet 1 (368)
- wallet 3 (34) ------/

They will be handled separately in the admin panel.

**Comment by ripzery** [comment]
I will split out the `/blockchain_wallet.create` to another issue. 🙇
